[BUGFIX] TGP: Beach Med Leaf-Bare (Part 3)

LaurBee opened the pull request:
<!-- Write BELOW The Headers and ABOVE The comments else it may not be viewable. -->
<!-- You can view CONTRIBUTING.md for a detailed description of the pull request process. -->
<!-- Be sure to name your PR something descriptive and succinct; include Bugfix: Feature: Enhancement: or Content: in the title to describe what type of PR it is. -->

## About The Pull Request
Finished pronounifying the Beach Med Leaf-Bare patrol file. Changes include:
- General text edits for grammar/better flow
- Pronoun/Verb tagging
- Adding more scars to outcomes
- Made stat outcomes more common
- Added a few new outcomes to some patrols
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage senior developers from merging your PR! -->

## Changelog/Credits
Patrol edits
<!-- Include any changes that should be made to the changelog of the game here or any changes to the credits file of the game. -->
<!-- This is just for easy access later for senior developers gathering this information; this process is not automated. -->

LaurBee committed:
TGP: Beach Med Leaf-Bare (Part 3)

Finished pronounifying the Beach Med Leaf-Bare patrol file

arisaema12 commented on resources/dicts/patrols/beach/med/leaf-bare.json:
```diff
@@ -2017,11 +2062,11 @@
         "success_outcomes": [
             {
-                "text": "p_l can't say it's <i>fun,</i> swiping away snow to scrounge for the stems and roots of wilted dandelions below, but what matters currently is that it's <i>possible.</i> {PRONOUN/p_l/subject/CAP} specifically ensure that, when {PRONOUN/p_l/subject} {VERB/p_l/return/returns} to camp with the additions to the herb stores, the whole camp sees {PRONOUN/p_l/subject} praising { ...
+                "text": "p_l can't say it's <i>fun,</i> swiping away snow to scrounge for the stems and roots of wilted dandelions below, but what matters currently is that it's <i>possible.</i> {PRONOUN/p_l/subject/CAP} specifically ensure that, when {PRONOUN/p_l/subject} {VERB/p_l/return/returns} to camp with the additions to the herb stores, the whole camp sees {PRONOUN/p_l/object} praising {P ...
```
```suggestion
                "text": "p_l can't say it's <i>fun,</i> swiping away snow to scrounge for the stems and roots of wilted dandelions below, but what matters currently is that it's <i>possible.</i> {PRONOUN/p_l/subject/CAP} specifically {VERB/p_l/ensure/ensures}  that, when {PRONOUN/p_l/subject} {VERB/p_l/return/returns} to camp with the additions to the herb stores, the whole camp sees {PRONOUN/p_l/object} praising {PRONOUN/p_l/poss} long-suffering patrol to the leader.",
```

LaurBee committed:
Update resources/dicts/patrols/beach/med/leaf-bare.json

Co-authored-by: saema <lucylouisecross@gmail.com>

LaurBee committed:
Merge branch 'ClanGenOfficial:development' into patrol-fixes

LaurBee committed:
Merge branch 'ClanGenOfficial:development' into patrol-fixes

scribblecrumb commented on resources/dicts/patrols/beach/med/leaf-bare.json:
```diff
@@ -1922,9 +1931,38 @@
+                ]
+            },
+            {
+                "text": "Not only does the duo end up soaked to the skin, they can't even find any dandelions under the snow. app1's left so exhausted that {PRONOUN/app1/subject} {VERB/app1/collapse/collapses} on the way back to camp, unable to go any further even with p_l's help..",
```
Is this meant to end with ... or with a single period?

LaurBee replied:
Whoops, must've missed that 😅 Single period. I'm away from my computer rn, but I'll get it fixed ASAP

LaurBee committed:
Merge branch 'ClanGenOfficial:development' into patrol-fixes

LaurBee committed:
Update leaf-bare.json

Fixed two instances where there was an incorrect amount of periods at the end of sentences
